### DISCUSSION TITLE
New release ?

### DISCUSSION THREAD

**Discussion by loickaczmarek** [GoogleChrome/lighthouse-ci#681] [Q&A]
Hello :)

I looked everywhere (I think) but I couldn't find any information about the releases. Is there one planned soon?

Thanks for your help !

**Comment by patrickhulce** [comment]
Information on releases can be found at https://github.com/GoogleChrome/lighthouse-ci/releases 

And yes good call there hasn't been one in a while despite many good updates :) I just shipped v0.8.1 👍
